### PR TITLE
Replace appcelerator relations

### PR DESCRIPTION
**Pull request by caspahouzer**
we have to clear this repo from Axway, Appcelerator Inc. and dead links

### CODE REVIEW

File: TitaniumKit.podspec
Comment (m1ga) [inline TitaniumKit.podspec:7]: Is this correct? Or just Titanium iOS...

File: TitaniumKit.podspec
Comment (m1ga) [inline TitaniumKit.podspec:10]: Is this correct? Or just Titanium iOS...

File: apidoc/Titanium/CHANGELOG/1.6.0.mdoc
Comment (m1ga) [inline apidoc/Titanium/CHANGELOG/1.6.0.mdoc:20]: keep the decription

File: apidoc/Titanium/CHANGELOG/1.6.0.mdoc
Comment (m1ga) [inline apidoc/Titanium/CHANGELOG/1.6.0.mdoc:40]: keep the decription

File: iphone/TitaniumKit/docs.sh
Comment (m1ga) [inline iphone/TitaniumKit/docs.sh:24]: Tidev?

File: iphone/TitaniumKit/docs.sh
Comment (m1ga) [inline iphone/TitaniumKit/docs.sh:25]: change the url?

File: tests/README.md
Comment (m1ga) [inline tests/README.md:3]: remove [Axway] from the link

**Comment by caspahouzer** [comment]
Can this be merged?

**Comment by cb1kenobi** [comment]
@caspahouzer nope. Nobody has approved it yet. I can review it, but then something else has to give.

**Comment by caspahouzer** [comment]
I don't understand that process 🙈
@m1ga has reviewed it and had commented what I then had fixed. 

Because of that I have asked

**Comment by hansemannn** [comment]
@m1ga Please use the "Approve" button under "Files changed" > "Review changes". Every PR should have an official review approval, so it's easier to see who managed the review process.

**Comment by m1ga** [comment]
@hansemannn I know, but I wasn't doing a full review with approval :smile: I was just quickly checking the files for any mistakes. I Since there are changes in some source files you should run the test suite with this SDK to make sure nothing breaks and I didn't do that so I didn't want to approve it.

**Comment by caspahouzer** [comment]
> @hansemannn I know, but I wasn't doing a full review with approval 😄 I was just quickly checking the files for any mistakes. I Since there are changes in some source files you should run the test suite with this SDK to make sure nothing breaks and I didn't do that so I didn't want to approve it.

Changes are only made in comments. Dare at last 😄

File: README.md
Comment (m1ga) [inline README.md:201]: broken link

File: apidoc/Titanium/CHANGELOG/2.1.0.GA.mdoc
Comment (m1ga) [inline apidoc/Titanium/CHANGELOG/2.1.0.GA.mdoc:281]: remove this line if the link below is gone

**Review by m1ga — COMMENTED** [review]
left some last comments. Test run fine, apps still build. So after that I can approve it from my side

**Review by m1ga — APPROVED** [review]
LGTM

**Pull request merged**
